src/lib.rs

```diff
@@ -35,6 +35,8 @@
 //! };
 //! ```
 //!
+//! # Important Notes before compiling!
+//!
```

Comment:
So I've put a header for people like me who rushes through the documentation to aware them of a few notes before compiling on Linux.